test/getAllDatasets-test.js

```diff
@@ -0,0 +1,15 @@
+const { expect } = require('chai');
+const { getAllDatasets } = require('../index');
+
+// Note: This test tests getAllDatasets and therefore implicitly tests getAllDatasetSiteUrls and extractJSONLDfromHTML.
+// In future, tests should test all public functions individually.
+describe('getAllDatasets()', function () {
+  it('should return all datasets', async () => {
+    const datasets = await getAllDatasets('https://openactive.io/data-catalogs/data-catalog-collection.jsonld', true);
```

Comment:
This will mean that CI is stalled if this URL is down for whatever reason. An alternative approach would be mocking axios with a snapshot of the data as it is now

Reply:
Yes good point - perhaps worth creating a mock `https://openactive.io/data-catalogs/data-catalog-collection.jsonld` that links to only a mock `https://openactive.io/data-catalogs/singular.jsonld` that links to only a mock Dataset Site of e.g. `https://reference-implementation.openactive.io/OpenActive` (which could be rendered using https://github.com/openactive/dataset-site-template#renderdatasetsitejsonld-staticassetspathurl using a copy of https://openactive.io/dataset-site-template/example.jsonld as input, which links to only `https://www.example.com/feeds/events`)